web/gui/src/dashboard.js/main.js

```diff
@@ -3160,6 +3160,19 @@ let chartState = function (element) {
         this.data_url += "&gtime=" + this.gtime;
         this.data_url += "&options=" + this.chartURLOptions();
 
+        if(NETDATA.options.redirect_freeze) {
```

Comment:
I didn't go into the details of the code to understand what's supposed to happen, but it didn't work for me. I click on the slack notification and I don't get centered on the correct time, or get the freezing.